### DISCUSSION TITLE
Is there anyway to use pg-promise with cloudflare workers?

### DISCUSSION THREAD

**Discussion by iamnottheway** [vitaly-t/pg-promise#926] [Q&A]
Hi! I've run into a bit of a snag. I came across pg-promise today while researching how to insert bulk records into postgres. I'm using Cloudflare workers to send data to the DB, but doing a single insert for many records would not be ideal. I tried using pg-promise when I deployed the worker I got this `TypeError: Cannot read properties of undefined (reading 'split')`. 

I looked into pg-promise internals and found that it checks for versions `process.versions.node.split('.'),` in the index file. I realize that Cloudflare workers don't support many nodejs packages but I was wondering if there's a way around it?

Thanks

**Comment by vitaly-t** [comment]
I'm not familiar with Cloudflare. Maybe somebody else will weight in.

But to start with, you can work around that check, to be ignored, and see what next issue, if any, it will produce ;)
